services/client/Dockerfile-dev

```diff
@@ -5,11 +5,11 @@ RUN mkdir /usr/src/app
 WORKDIR /usr/src/app
 
 # install and cache app dependencies
-ADD package.json package-lock.json /usr/src/app/
+COPY package.json package-lock.json /usr/src/app/
 RUN cd /usr/src/app && npm install --silent
 
 # add app
-# ADD . /usr/src/app
+COPY . /usr/src/app
```

Comment:
Uncommented this `ADD` command, and changed to `COPY` which is a little less magical (TIL)